addons/voice/pom.xml

```diff
@@ -20,6 +20,7 @@
     <!--<module>org.openhab.voice.kaldi</module>-->
     <module>org.openhab.voice.marytts</module>
     <module>org.openhab.voice.voicerss</module>
+    <module>org.openhab.voice.pico2wave</module>
```

Comment:
Please add at the correct alphabetical place.

Reply:
I assume this should also be added to the feature.xml please check whether/where the others are included

Reply:
Done! :)